README.md

```diff
@@ -4,7 +4,108 @@
+Foundry makes creating fixtures data fun again, via an expressive, auto-completable, on-demand fixtures system with
+Symfony and Doctrine:
+
+```php
+$post = PostFactory::new() // Create the factory for Post objects
+    ->published()          // Make the post in a "published" state
+    ->create([             // create & persist the Post object
+        'slug' => 'post-a' // This test only requires the slug field - all other fields are random data
+    ])
+;
+```
+
+The factories can be used inside [DoctrineFixturesBundle](https://symfony.com/doc/master/bundles/DoctrineFixturesBundle/index.html)
+to load fixtures or inside your tests, [where it has even more features](#using-in-your-tests).
+
+## Documentation
+
+1. [Installation](#installation)
+2. [Using with DoctrineFixturesBundle](#using-with-doctrinefixturesbundle)
+3. [Using in your Tests](#using-in-your-tests)
+4. [Sample Entities](#sample-entities)
+5. [Model Factories](#model-factories)
+    1. [Generate](#generate)
+    2. [Usage](#usage)
+    3. [States](#states)
+    4. [Attributes](#attributes)
+    5. [Faker](#faker)
+    6. [Events](#events)
+    7. [Instantiation](#instantiation)
+    8. [Initialization](#initialization)
+    9. [Immutable](#immutable)
+    10. [Doctrine Relationships](#doctrine-relationships)
+    11. [Anonymous Factories](#anonymous-factories)
+6. [Stories](#stories)
+    1. [Stories as Services](#stories-as-services)
+7. [Testing](#testing)
+    1. [Auto-Refresh & Force Setting](#auto-refresh--force-setting)
+    2. [Repository](#repository)
+    3. [Assertions](#assertions)
+    4. [Global State](#global-state)
+    5. [Performance](#performance)
+        1. [DAMADoctrineTestBundle](#damadoctrinetestbundle)
+        2. [Miscellaneous](#miscellaneous)
+    6. [Using without the Bundle](#using-without-the-bundle)
+8. [Full Default Bundle Configuration](#full-default-bundle-configuration)
+9. [Credit](#credit)
+
+## Installation
+
+    $ composer require zenstruck/foundry --dev
+
+To use the *Maker's*, ensure [Symfony MakerBundle](https://symfony.com/doc/current/bundles/SymfonyMakerBundle/index.html)
+is installed.
+
+*If not using Symfony Flex, be sure to enable the bundle in your **test**/**dev** environments.*
+
+## Using with DoctrineFixturesBundle
+
+Foundry works out of the box with [DoctrineFixturesBundle](https://symfony.com/doc/master/bundles/DoctrineFixturesBundle/index.html).
+You can simply use your factory's and story's right within your fixture files:
+
+```php
+// src/DataFixtures/AppFixtures.php
+namespace App\DataFixtures;
+
+use App\Factory\CategoryFactory;
+use App\Factory\PostFactory;
+use App\Factory\TagFactory;
+use App\Story\GlobalStory;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+
+class AppFixtures extends Fixture
+{
+    public function load(ObjectManager $manager)
+    {
+        GlobalStory::load();
```

Comment:
I moved to the bottom but...

Maybe it should be removed entirely?

I named it `GlobalStory` to match what I talk about in the global test state section: apps (mine anyway) may require a specific set of *default* data in their database to function property. This could be packaged up into a `GlobalStory` and loaded before your tests and fixtures.

So.. having at the top, shows that this *needs to be loaded first*.

Reply:
ooook. I think it shouldn't be mentioned here (wtf is a global story? Is that a core class from this library?). You already have this `GlobalStory` idea mentioned in "Global State"... I think that is enough for now.

Reply:
4ddcb07ddf45836d3d4f12d794708250e48d4653